keras/backend/tensorflow_backend.py

```diff
@@ -1319,6 +1319,7 @@ def switch(condition, then_expression, else_expression):
         condition: scalar tensor.
         then_expression: TensorFlow operation.
         else_expression: TensorFlow operation.
+        lazy: Unused (compatibility with Theano backend)
```

Comment:
Not sure what you mean, but the purpose here was to ensure that we don't get a ```TypeError``` upon switching implementations and using the ```lazy``` kwarg to ```switch```.